entries/k/klantportaal.digireceptie.nl.json

```
@@ -0,0 +1,11 @@
+{
+  "DigiReceptie Klantportaal": {
+    "domain": "klantportaal.digireceptie.nl",
+    "tfa": [
+      "totp"
+    ],
+    "keywords": [
+      "security"
```

I think `other` is a better category here since the service doesn't really fit any of our existing categories.
```suggestion
      "other"
```